minecraft/networking/packets/clientbound/play/map_packet.py

```diff
@@ -100,7 +101,10 @@ def read(self, file_object):
             icon = MapPacket.MapIcon(type, direction, (x, z), display_name)
             self.icons.append(icon)
 
-        self.width = UnsignedByte.read(file_object)
+        try:
+            self.width = UnsignedByte.read(file_object)
+        except:
+            self.width = None
```

Comment:
What is the reason for this? If the `width` read was raising an exception, I feel like there's probably a better way of dealing with that.

Reply:
Ya that was me be young and lazy to find out the issue when I played on a earlier mc version. I intend to fix that as that is a situational patch that plagued me on a server that I played on.

Reply:
Maybe the packet format is being parsed incorrectly for earlier versions. Do you remember which version it was?

Reply:
Let me get back to you on that as this was mainly a fix and forget type of situation as it seems like a random inconsistent bug. I can play for hours back then and the bug wouldn't show up. I suspect the server I played on might be sending malformed packets or something to that nature.